Experiment_Launcher_code/ExperimentManager.py

```diff
@@ -25,19 +28,25 @@ def __init__(self, comport):
         # initialize software components
         self.reward_manager = RewardManager(comport)
         self.videoAnalyser = Video_Analyzer()
+        self.mouse_id="1780"
+        self.videoAnalyser = Video_Analyzer(self.mouse_id)
         self.stateManager = StateManager()
         self.trial_logger = TrialLogger()
+        self.stateHistories = {}
         #initialize data_analyser
         #self.data_analyzer = DataAnalyzer(self.data_path)
-        #initialize experimenter
-        #self.experimenter = Experimenter(self.videoAnalyser)
+        # Initialize Arduino board
```

Comment:
Initializing of Arduino and reward manager are not needed. Reward manager is initialized a few lines earlier, and it will initialize the Arduino itself.